tests/openstack/spread.yaml

```diff
@@ -0,0 +1,50 @@
+project: spread
+
+backends:
+    openstack:
+        plan: m1.tiny
+        halt-timeout: 1s
+        systems:
+            - cirros-64:
+                  image: cirros-0.6.2-x86_64-disk
+                  groups:
+                        - default
+                  networks:
+                        - private
+            - cirros-64-wrong-network:
+                  image: cirros-0.6.2-x86_64-disk
+                  groups:
+                        - default
+                  networks:
+                        - noexist
```

Comment:
"noexist" no exist in English. :smile:

"invalid" works better and is more clear. There are more occurrences.

Reply:
done